frontend/src/employee-frontend/components/vasu/templates/VasuTemplateEditor.tsx

```diff
@@ -65,6 +66,16 @@ export default React.memo(function VasuTemplateEditor() {
 
   const readonly = !(template.isSuccess && template.value.documentCount === 0)
 
+  const translations = useMemo(
+    () =>
+      template
+        ? (vasuTranslations[
+            template.getOrElse({ language: 'FI' }).language.toLowerCase()
+          ] as VasuTranslations)
+        : vasuTranslations.fi,
```

Comment:
`template` is always defined so the ternary is redundant. How about 

```ts
vasuTranslations[
    template.map((t) => t.language.toLowerCase() as Lang).getOrElse('fi')
]
```
Would be even better if we had a type-safe `lang` to `vasu lang` mapping fn somewhere.